app/src/main/java/com/garethevans/church/opensongtablet/MenuHandlers.java

```diff
@@ -76,10 +75,8 @@ static void forceOverFlow(Context c, ActionBar ab, Menu menu) {
             ViewConfiguration config = ViewConfiguration.get(c);
             if (Build.VERSION.SDK_INT<Build.VERSION_CODES.R) {
                 Field menuKeyField = ViewConfiguration.class.getDeclaredField("sHasPermanentMenuKey");
-                if (menuKeyField != null) {
-                    menuKeyField.setAccessible(true);
-                    menuKeyField.setBoolean(config, false);
-                }
+                menuKeyField.setAccessible(true);
+                menuKeyField.setBoolean(config, false);
```

Comment:
This is the only bit of the pull I'm a bit wary of.  This is a 'hacky' way to check for hardware back buttons on phones and from memory, it quite regularly throws a null menuKeyField value as not all versions of Android have this declaredField.  Every example online has the null check before setting values. 
Given that most devices don't have hardware devices, it is probably more sensible to either leave the null check, or remove this check altogether (as I have in v6).

Reply:
Code Inspector suggests that this is not needed - but what does it know! I have reverted. :-)